x-pack/plugins/apm/public/components/app/service_overview/service_overview_throughput_chart.tsx

```diff
@@ -111,20 +118,43 @@ export function ServiceOverviewThroughputChart({
 
   return (
     <EuiPanel hasBorder={true}>
-      <EuiTitle size="xs">
-        <h2>
-          {i18n.translate('xpack.apm.serviceOverview.throughtputChartTitle', {
-            defaultMessage: 'Throughput',
-          })}
-        </h2>
-      </EuiTitle>
+      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xs">
+            <h2>
+              {i18n.translate(
+                'xpack.apm.serviceOverview.throughtputChartTitle',
+                { defaultMessage: 'Throughput' }
+              )}
+            </h2>
+          </EuiTitle>
+        </EuiFlexItem>
+
+        <EuiFlexItem grow={false}>
+          <EuiIconTip
+            content={
+              data.throughputUnit === 'minute'
+                ? i18n.translate('xpack.apm.serviceOverview.tpmHelp', {
+                    defaultMessage:
+                      'Throughput is measured in tpm (transactions per minute)',
+                  })
+                : i18n.translate('xpack.apm.serviceOverview.tpsHelp', {
+                    defaultMessage:
+                      'Throughput is measured in tps (transactions per second)',
```

Comment:
Maybe you could inform the user when we break from `tpm` to `tps`?!

Reply:
Yes, I thought about it but didn't want it to be too confusing. So rather than explaining what both tpm and tps is, it just explains the current state.

WDYT @formgeist

Reply:
Not sure we need to clarify when we switch. I would suggest only to clearly show a different measurement when we switch to seconds. The default measure is minutes, which we can still show in a tooltip next to the title. But we'll change the title and tooltip when we switch, like so;

![throughput](https://user-images.githubusercontent.com/4104278/128820732-b08b6c68-c48e-4c6e-9890-dd3cbfa9df4d.png)

Reply:
@formgeist I added the `(per second)` label